Veriff/Detector/Detector.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Detector.swift
+//  Veriff
+//
+//  Created by Kseniya Zharikova on 27/7/22.
+//
+
+import Vision
+import UIKit
+
+final class Detector {
+    
+    func detectFace(image: UIImage, _ completion: @escaping (Result<UIImage, Veriff.ErrorReason>) -> Void) {
+        let request = VNDetectFaceRectanglesRequest { (request, error) in
+            if let _ = error {
+                completion(.failure(.internalError))
+            }
+            
+            guard let results = request.results else {
+                completion(.failure(.internalError))
+                return
+            }
+            
+            if results.count == .zero {
+                completion(.failure(.noFace))
+            } else if results.count > 1 {
+                completion(.failure(.moreThanOneFace))
+            } else {
+                completion(.success(image))
+            }
+        }
+        
+        detect(image: image, reguest: request)
+    }
+    
+    func detectText(image: UIImage, _ completion: @escaping (Result<[String], Veriff.ErrorReason>) -> Void) {
+        let request = VNRecognizeTextRequest { (request, error) in
```

Comment:
```suggestion
        let request = VNRecognizeTextRequest { request, error in
```

по стилю всегда лучше придерживаться минимализма, поэтому если язык позволяет что-то не писать (например, точку с запятой или скобки), то лучше не писать